components/integration/base/delete-snapshots.yaml

```
@@ -0,0 +1,26 @@
+kind: ClusterRole
+apiVersion: authorization.openshift.io/v1
+metadata:
+  name: delete-snapshots
+rules:
+  - apiGroups:
+    - "appstudio.redhat.com"
+    resources:
+      - "snapshots"
+    verbs:
+      - "delete"
+      - "deletecollection"
+---
+kind: RoleBinding
```

If the intent is to allow integration team to delete snapshots in all namespaces as mentioned in the commit msg/PR description, you need to do a ClusterRoleBinding otherwise a RoleBinding as you did will only allow to delete snapshots in integration-service namespace.

```suggestion
kind: ClusterRoleBinding
```